sw/device/lib/dif/dif_spi_device.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "SPI DEVICE DIF Checklist"
```

Comment:
I do love that this renders as "SPI DEVICE".

Reply:
Yeah that's why I updated the template… fixed now.